tests/javascript/utils/matchTypes.test.ts

```diff
@@ -0,0 +1,133 @@
+/**
+ * @copyright Copyright (c) 2023 Daniel Kesselberg <mail@danielkesselberg.de>
+ *
+ * @author Daniel Kesselberg <mail@danielkesselberg.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { matchTypes } from '../../../src/utils/matchTypes'
+import rfcProps from '../../../src/models/rfcProps.js'
+
+describe('utils/matchTypes test suite', () => {
+
+	describe('impp', () => {
+		it('match', () => {
```

Comment:
```suggestion
		it('matches', () => {
```

if you want your test cases to read like English sentences :)